### DISCUSSION TITLE
Email Plugin Settings Not Displayed

### DISCUSSION THREAD

**Discussion by greyvity** [strapi/strapi#7071] [Legacy Topics]
I installed a strapi app using the command
`npx create-strapi-app name --quickstart`
**Everything else seems to be working correctly but I cannot find any settings icon in the email plugin. Any other part of the plugins page is not clickable except the download and delete options in the screenshot.**  
And I haven't made any changes to the code. Just the out of the box strapi app. I added a couple of content-types but irrespective of that, the issue still exists.
I tried creating a couple of more strapi apps but they all had this same issue.

_versions: 
strapi: 3.0.6
node: v12.18.2
npm: 6.14.6
browsers: chrome and firefox_

![image](https://user-images.githubusercontent.com/57741151/87865589-fa4f4d80-c996-11ea-8204-41bea9dc82c9.png)
![image](https://user-images.githubusercontent.com/57741151/87865597-118e3b00-c997-11ea-971b-bbdf3f241460.png)

**Comment by oritwoen** [comment]
In the current version of Strapi, email settings are defined by an appropriate file. Take a look at the documentation: https://strapi.io/documentation/v3.x/plugins/email.html#configure-the-plugin

**Reply by greyvity** [reply]
Thank you,
I am new to the Strapi community.
My bad. I didn't understand the docs well.
I used strapi-provider-email-nodemailer.
It works now.
Thanks a bunch :v
